src/options.ts

```diff
@@ -1,7 +1,9 @@
 export interface TOptions {
   maxSize?: number;
+  type?: string;
```

Comment:
I think it's better like:

```ts
type?: 'image/png' | 'image/jpeg' | 'image/webp';
```

Because maybe dose not work, if [other mime type](https://www.iana.org/assignments/media-types/media-types.xhtml) is used🤔

Reply:
Good catch, done :)